container/run.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env
+
+export SUPPRESS_SUPPORT=1
+export KEG_CONSUMER_BUILD_PATH=/keg/app/@keg-hub/
```

Comment:
another question. this `KEG_CONSUMER_BUILD_PATH` is defined both here and the `values.yml` but the values are different. 
I see that it's being used inside `config-overrides.js`, but which value is it using?

Reply:
If it's defined here, and `run.sh` is used to start the consume, then it's using the value here. 
If it's run from something like `keg consume att`, then `yarn start`; then it would use the value form the values.yml file

Reply:
yep this was not intended to be here anymore. 

I have removed it in the latest commits